readthedocs/projects/forms.py

```diff
@@ -588,6 +589,9 @@ class Meta:
             "doc_diff_enabled",
             "external_version_warning_enabled",
             "flyout_enabled",
+            "flyout_sorting",
+            "flyout_sorting_latest_stable_at_beginning",
+            "flyout_sorting_custom_pattern",
```

Comment:
Are we doing any logic to make this show/hide based on the value? I don't see it in this PR.

Reply:
Yes. We will need to work on this in the front-end (ext-theme) to show/hide them dynamically using KO based on what the user selects on flyout sorting.